src/system/obj/Data.h

```diff
@@ -119,10 +121,16 @@ class DataNode {
     ~DataNode();
 
     DataType Type() const { return mType; }
-    DataNodeValue RawVal() { return mValue; }
     bool CompatibleType(DataType) const;
     DataNode& Evaluate() const;
 
+    // these were implemented to match up in retail
+    // please do not use these in regular code
+    int FastInt() const { return mValue.integer; }
+    const char* FastStr() const { return mValue.symbol; }
+    Hmx::Object* FastObj() const { return mValue.object; }
+    DataArray* FastArray() const { return mValue.array; }
```

Comment:
I'd say these should be labelled as `Unchecked` rather than `Fast`, to better indicate the intended use case.

Reply:
Good idea, I think I'll rename it to `Unsafe` to indicate that it's, well, unsafe, and you shouldn't use these ordinarily lol

Reply:
pushed

Reply:
That works too :+1:
I've just typically seen `Unchecked` used for this kind of thing, to imply that you are responsible for doing the necessary checks up-front.

Reply:
since Unchecked is what's typically used, I went ahead and renamed that to match common conventions